### ISSUE TITLE
Line length does not correctly evaluates the relative path from moduleroot (or source root)

### ISSUE THREAD

**Issue by johlju** [dsccommunity/DscResource.Test#37]
```
      [-] The length of the relative full path C:\source\xWebAdministration\source\DSCResources\MSFT_xWebConfigPropertyCollection\en-US\MSFT_xWebConfigPropertyCollection.strings.psd1 should not exceed the max hard limit 3ms
        Expected the actual value to be less than or equal to 129, but got 135.
        49:             $FullRelativePath.Length | Should -Not -BeGreaterThan $fullPathHardLimit
        at <ScriptBlock>, C:\source\xWebAdministration\output\RequiredModules\DscResource.Test\0.8.0\Tests\QA\RelativePathLength.common.Tests.ps1: line 49
```

**Comment by PlagueHO** [comment]
Ah yes, I got bitten by this one in xPSDesiredStateConfiguration - so would be good to get fixed.

**Comment by dan-hughes** [comment]
@johlju, @PlagueHO, Is this still an issue? I think this is fixed by #35. I cloned WebAdministrationDsc and the Relative Path Length tests passed.

**Comment by johlju** [comment]
I think the the path length is less not when WebAdministrationDsc has removex the 'x' prefix and chnage MSFT_ to DSC_ 🤔  Could have been fixed. Let us close it and reopen if it is hit somewhere else. 🙂
